WordPress/openverse-frontend
# Replace download button with foreign landing button

## Fixes


Fixes #1250 by @zackkrida 

## Description


As agreed in [this comment](https://github.com/WordPress/openverse-frontend/issues/1250#issuecomment-1100136519) this PR replaces the audio format download button with a link button to the foreign landing URL for the track.

I also deleted the unused dropdown and download button components. They're not exactly implemented in a way that I think we would implement them today (I was still very much learning Vue when I wrote them) and we can write better ones if we decide we need this again (or just restore them from the git history if we don't have time).

## Testing Instructions

Checkout the branch and run `pnpm dev`. Visit an audio track like `/audio/3b95bd5f-55fe-4b6f-be83-26a3e5886674` and check the download button. Visit another and make sure it works :slightly_smiling_face: 

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
